www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -30,278 +30,676 @@ platform-centered shell tools or cross-platform Cordova CLI for
 development. For a comparison of the two development paths, see the
 [Overview](../../overview/index.html).  For details on the CLI, see [The Command-Line Interface](../../cli/index.html).
```

Comment:
Link should point to: http://localhost:3000/docs/en/dev/guide/overview/index.html#development-paths